### PR TITLE
[V2] Yocto BSP v5.9 kirkstone + Linux DDK v1.15 + other misc things

### PR DESCRIPTION
**Pull request by otyshchenko1**

### CODE REVIEW

**Comment by otyshchenko1** [comment]
This is V2 for https://github.com/xen-troops/meta-xt-prod-devel-rcar/pull/80 The WIP mask was lifted.

File: prod-devel-rcar.yaml [hunk 1]
```diff
@@ -109,6 +109,12 @@ common_data:
     # Install some apps to test GPU/Display
     - [IMAGE_INSTALL:append, " glmark2 kmscube"]
 
+    # Do not mask bbappends (which purpose is to switch back to old DDK v1.11) in case of DomA build
+    # mask the recipes in order to use new DDK v1.15 for DomU build
+    - [BBMASK:append, " ${@bb.utils.contains('XT_GUEST_INSTALL', 'doma', '', 'meta-xt-rcar-fixups/recipes-kernel/kernel-module-gles/kernel-module-gles.bbappend', d)}"]
```
Comment (lorc) [inline prod-devel-rcar.yaml:114]: Should we rely on presence of `doma`? Maybe there is a better way?

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by otyshchenko1** [comment]
> `Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

Thanks!

**Comment by otyshchenko1** [comment]
updated, only added A-b

**Review by lorc — APPROVED** [review]

**Pull request merged**